src/wic/api/_types.py

```diff
@@ -0,0 +1,30 @@
+"""CWL Types."""
+from enum import Enum
+from pathlib import Path
+
+
+class CWLTypesEnum(str, Enum):
+    NULL = "null"
+    BOOLEAN = "boolean"
+    INT = "int"
+    LONG = "long"
+    FLOAT = "float"
+    DOUBLE = "double"
+    STRING = "string"
+    FILE = "File"
+    DIRECTORY = "Directory"
+
+
+CWL_TYPES_DICT: dict[str, object] = {
+    "null": None,
+    "boolean": bool,
+    "int": int,
+    "long": int,
+    "float": float,
+    "double": float,
+    "string": str,
+    # "File": CWLFile,
+    # "Directory": CWLDir,
```

Comment:
Could you please remove duplicated keys and values from CWL_TYPES_DICT dictionary?

Reply:
Addressed in #135